launchable/commands/record/session.py

```diff
@@ -82,6 +82,14 @@ def _validate_session_name(ctx, param, value):
     metavar='SESSION_NAME',
     callback=_validate_session_name,
 )
+@click.option(
+    '--lineage',
+    'lineage',
+    help='Set lineage name. A lineage is a set of test sessions grouped and this option value will be used for grouping.',
```

Comment:
`this option value will be used for grouping.` 
Do you mean the lineage name becomes group name, or something else?

Reply:
Ah... I should fix `this option value will be used for a lineage name`